pinot-core/src/main/java/com/linkedin/pinot/core/segment/creator/ColumnStatistics.java

```diff
@@ -48,7 +48,13 @@
 
     /**
      *
-     * @return For string objects, returns the length of the longest string value. For others, returns -1.
+     * @return For variable length objects, returns the length of the smallest value. For others, returns -1.
+     */
```

Comment:
Should be shortest element, not smallest.

Reply:
Will change.